setup.py

```diff
@@ -10,7 +10,7 @@
     long_description_content_type='text/markdown',
     version='1.0.5',
     packages=find_packages(),
-    install_requires=["natsort==7.0.1"],
+    install_requires=["natsort==7.0.1", "pyfakefs", "mock"],
```

Comment:
I think these should be put in `tests_require` instead. And it would be better if you could specify package version to prevent errors when there're breaking changes.

Reply:
Done. I chose the latest version for pyfakefs and the latest stable version for mock (the next version, 4.0.0, causes an error for some of the unit tests).